scripts/build-crossdock.sh

```diff
@@ -21,10 +21,7 @@ else
   exit 0
 fi
 
-docker build -f crossdock/Dockerfile \
-	--build-arg tornado=$TORNADO \
-	--tag $REPO:$COMMIT .
-
+# This image is already built in `make crossdock` step
+docker tag crossdock_python:latest $REPO:$COMMIT
```

Comment:
Q: where is the name `crossdock_python:latest` coming from? It seems some implicit default behavior of docker-compose. If so, is it possible to give it a name explicitly instead (see next comment for why)?

Reply:
Right it seems some implicit behaviour (crossdock is the directory name where docker-compose file exists and python is the service name within that docker_compose file). There should be some way to name it explicitly. Are you thinking about situation when changing something in docker-compose cause this script to fail since things are implicit.

Reply:
Yes, but I am not sure it would work even now, because when pushing you're referring to a different name altogether.

If docker-compose does not allow to override the name, we could just add an explicit build step to `make crossdock` and refer to that image from the docker-compose.yml

Reply:
From: https://docs.docker.com/compose/reference/build/

> If the Compose file specifies an image name, the image is tagged with that name

Hence, adding the image name in docker-compose should build image with that name.